Bottom: 5420e097243dcce7eadf5c6cfce1fcdc4ce22bc3
Top:    594d91eca00ab4ce0ccfe6afa94446913c966da3
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:42:02 -0500

unique_ptr warning


---

diff --git a/libstdc++-v3/include/bits/unique_ptr.h b/libstdc++-v3/include/bits/unique_ptr.h
index 3695214808b..3062c62d6ab 100644
--- a/libstdc++-v3/include/bits/unique_ptr.h
+++ b/libstdc++-v3/include/bits/unique_ptr.h
@@ -464,8 +464,8 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
       }
 
       // Disable copy from lvalue.
-      unique_ptr(const unique_ptr&) = delete;
-      unique_ptr& operator=(const unique_ptr&) = delete;
+      unique_ptr(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
+      unique_ptr& operator=(const unique_ptr&) = delete; // If you see an error here, you probably forgot to use std::move with unique_ptr
   };
 
   /// 20.7.1.3 unique_ptr for array objects with a runtime length
